pkg/catalog/debugger.go

```diff
@@ -10,10 +10,10 @@ import (
 
 // ListSMIPolicies returns all policies OSM is aware of.
 func (mc *MeshCatalog) ListSMIPolicies() ([]*split.TrafficSplit, []identity.K8sServiceAccount, []*spec.HTTPRouteGroup, []*access.TrafficTarget) {
-	trafficSplits := mc.meshSpec.ListTrafficSplits()
-	serviceAccounts := mc.meshSpec.ListServiceAccounts()
-	trafficSpecs := mc.meshSpec.ListHTTPTrafficSpecs()
-	trafficTargets := mc.meshSpec.ListTrafficTargets()
+	trafficSplits := mc.Interface.ListTrafficSplits()
```

Comment:
since interface is embedded, you don't need to call it. I think `mc.ListTrafficSplits()` is a cleaner read, and makes it more easy for us to merge these 2 in the future.